lib/ExpensiMark.js

```diff
@@ -121,7 +126,33 @@ export default class ExpensiMark {
             },
 
             /**
-             * Converts markdown style images to img tags e.g. ![Expensify](https://www.expensify.com/attachment.png)
+             * Converts markdown style video to video tags e.g. ![Expensify](https://www.expensify.com/attachment.mp4)
+             * We need to convert before image rules since they will not try to create a image tag from an existing video URL
+             * Extras arg could contain the attribute cache for the video tag which is cached during the html-to-markdown conversion
+             */
+            {
+                name: 'video',
+                regex: MARKDOWN_VIDEO_REGEX,
+                /**
+                 * @param {string} match
+                 * @param {string} g1 - The first capture group - video name
```

Comment:
Please rename these to `group1` and `group2`. We should never have allowed other authors to get away with it in the other functions because it's against our style guide. Bonus points if you update the other method params so everything is consistent!